edx_proctoring/api.py

```diff
@@ -892,7 +892,7 @@ def _create_and_decline_attempt(exam_id, user_id):
     )
 
 
-def _register_proctored_exam_attempt(user_id, exam_id, exam, attempt_code, review_policy, verified_name=None):
+def _register_proctored_exam_attempt(user_id, exam_id, exam, attempt_code, review_policy):
```

Comment:
Why do we not want to use a verified name for registering a proctored exam attempt anymore?

Reply:
I think there are no callers now which set verified enabled which means this name was never populated.

Possibly this PR points to a completely different fix which does not go this direction.